app/controllers/api/v1/websites_controller.rb

```diff
@@ -4,7 +4,8 @@ def index
   end
 
   def create
-    @website = Website.create(domain: website_params[:domain], title: website_params[:title], member_id: current_member.id)
+    @website = Website.create( domain: website_params[:domain], title: website_params[:title], member_id: current_member.id )
+    WebsiteMember.create( member_id: current_member.id, website_id: @website.id, role: 1 )
```

Comment:
la fel si aici `@website.members.create(member_id: current_member.id, role: :owner)`  aici role nu stiu daca merge asa dar as prefera sa mearga decat sa imi setezi tu 1 acolo. In cel mai rau caz poti sa faci:
`@website.members.create(member_id: current_member.id, role: Member.role.find_value(:owner).value)` ne folosim deci de enumerize gem.